README.md

```diff
@@ -79,7 +79,9 @@ Highlights:
 ## Monitor
 
 - Once flashed, the board can be connected with any suitable serial monitor, e.g.:
-  - (Recommended) `espflash`: `espflash serial-monitor`
+  - ESPMonitor: `espmonitor /dev/ttyUSB0` (you need to `cargo install espmonitor` first)
+  - Cargo PIO (this one **decodes stack traces**!): `cargo pio espidf monitor /dev/ttyUSB0` (you need to `cargo install cargo-pio` first)
```

Comment:
Is `cargo-pio` "official" way to see stack traces?

Reply:
I don't know what happened here. Just created a fresh fork 20 min ago and committed changes on the typo and AP IP address only. I have no changes/opinion on `cargo-pio`.

Reply:
> I don't know what happened here. Just created a fresh fork 20 min ago and committed changes on the typo and AP IP address only. I have no changes/opinion on `cargo-pio`.

What happened is that you did not pull. The '-' ("Recommended") change is the new thing, which you somehow reverted.

Reply:
Normally you don't have to 'pull' after a 'clone' but I now understand what happened. To submit the change I had to first create a fork, which I did this morning. Since I already made the change in the clone from a few days ago I just copy-pasted the README.md to the fork.  I should have 'pull'-ed the old clone first before this copy action or just fix the README.md in the fresh fork which did include your recent change.

Reply:
I get it but I'm overwhelmed. Can you update your PR please so that I don't have to apply manual changes mtself? Thanks a lot!

Reply:
Done.